crypt.go

```diff
@@ -193,20 +207,64 @@ func (s *activateWithKeyDataState) run() (success bool) {
 			continue
 		}
 
-		return true
+		return true, nil
 	}
 
-	// TODO: Passphrase support
+	// Try keys that require a passphrase
+	tries := s.passphraseTries
+	var lastErr error
```

Comment:
wouldn't passphraseError be a  clearer name for this?

Reply:
Yeah, that sounds ok